src/chip.cs

```diff
@@ -1764,4 +1800,706 @@ interface ChipQRCodeSetupPayloadParser
 		ChipSetupPayload PopulatePayload ([NullAllowed] out NSError error);
 	}
 
+	[Mac (12,0), Watch (8,0), TV (15,0), iOS (15,0), MacCatalyst (15,0)]
+	[BaseType (typeof (ChipCluster), Name="CHIPAccountLogin")]
+	[DisableDefaultCtor]
+	interface ChipAccountLogin
+	{
+
+		[Export ("initWithDevice:endpoint:queue:")]
+		[DesignatedInitializer]
+		IntPtr Constructor (ChipDevice device, byte endpoint, DispatchQueue queue);
+
+		[Async (ResultTypeName = "ChipReadAttributeResult")]
+		[Export ("getSetupPIN:responseHandler:")]
+		void GetSetupPin (string tempAccountIdentifier, ChipResponseHandler responseHandler);
+
+		[Async (ResultTypeName = "ChipReadAttributeResult")]
+		[Export ("login:setupPIN:responseHandler:")]
+		void Login (string tempAccountIdentifier, string setupPin, ChipResponseHandler  responseHandler);
```

Comment:
```suggestion
		void Login (string tempAccountIdentifier, string setupPin, ChipResponseHandler responseHandler);
```
small: extra space here and next selector